website/content/docs/sync/gcpsm.mdx

```diff
@@ -1,7 +1,160 @@
 ---
 layout: docs
-page_title: GCP Secret Manager - Secrets Sync Destination
-description: The GCP Secret Manager destination syncs secrets from Vault to GCP.
+page_title: Google Cloud Platform Secret Manager - Secrets Sync Destination
+description: The Google Cloud Platform Secret Manager destination syncs secrets from Vault to GCP.
 ---
 
-# GCP Secret Manager
+# Google Cloud Platform Secret Manager Sync Destination
```

Comment:
A [recent change](https://github.com/hashicorp/vault/pull/21426) is to use sentence case for all titles. I think we want this to be?
```suggestion
# Google Cloud Platform Secret Manager sync destination
```

Reply:
Thanks, didn't know about this!